.env.example

```diff
@@ -1,2 +0,0 @@
-# RENAME TO .env AND POPULATE
-GOV_NOTIFY_API_KEY=
```

Comment:
Can we restore this? It's meant to be committed to the repo as a reference

Reply:
Restored

Reply:
Soz renamed and therefore deleted in error